test/catalog_test.exs

```diff
@@ -15,17 +15,18 @@ defmodule Wanda.CatalogTest do
     test "should return the whole catalog" do
       catalog_path = Application.fetch_env!(:wanda, Wanda.Catalog)[:catalog_path]
 
-      files =
+      valid_files =
         catalog_path
         |> File.ls!()
         |> Enum.sort()
+        |> Enum.filter(fn file -> file != "malformed_check.yaml" end)
```

Comment:
You don't really need to do this. Actually, is nice to see that the catalog doesn't return malformed checks

Reply:
We need to do this in order for the assertions in this test to be correct:

```Elixir
assert length(valid_files) == length(catalog)
```

and

```Elixir
Enum.with_index(catalog, fn check, index ->
        file_name =
          valid_files
          |> Enum.at(index)
          |> Path.basename(".yaml")

        assert %Check{id: ^file_name} = check
end)
```

Reply:
I see. Ok them. It looks ugly to have the name of the malformed check there hardcoded, but it is what it is hehe